### ISSUE TITLE
Sprite Inconsistency / Missing Sprites

### ISSUE THREAD

**Issue by sora10pls** [msikma/pokesprite#63]
### Case 1: Hitmonlee
Hitmonlee had its sprite updated in SWSH, it does not use the same sprite as the one present in LGPE. Its Shiny sprite should be updated to reflect the sprite change for consistency.

Normal (SWSH sprite):
![normal](https://user-images.githubusercontent.com/17801814/76688894-93481380-6607-11ea-9120-738dfff4d9a3.png)
Shiny (based on LGPE sprite):
![shiny](https://user-images.githubusercontent.com/17801814/76688895-93e0aa00-6607-11ea-8e3d-960205af3722.png)

### Case 2: Stonjourner
Stonjourner is missing a Shiny sprite entirely. Both sprites present in the project match the vanilla sprite found in SWSH.

**Comment by msikma** [comment]
Thanks for reporting. I actually noticed the difference for Hitmonlee and remade its shiny after finishing the LGPE models, but I seem to have messed up when I put the project together and used the wrong one.

For Stonjourner, I messed up embarrassingly. I made its shiny sprite, but I always keep a non-shiny layer in the PSD file for reference, and I accidentally had that layer enabled when I saved its png file. Here's the shiny sprite, I'll add it to the project:

![poke_icon_0974_00s_na](https://user-images.githubusercontent.com/1507745/76785540-ed86d700-67b5-11ea-8878-9f36f8c7bf57.png) ![poke_icon_0974_00s_n](https://user-images.githubusercontent.com/1507745/76785546-efe93100-67b5-11ea-9f85-da692e240b73.png)
